### PR TITLE
:sparkles: v3 (feature): add CHIPS support to Cookie

### PR DESCRIPTION
**Pull request by efectn**
## Description
Add support for partitioned cookies.

Fixes https://github.com/gofiber/fiber/issues/3002

## Changes introduced
- [x] Documentation Update: Detail the updates made to the documentation and links to the changed files.
- [x] Changelog/What's New: Include a summary of the additions for the upcoming release notes.

## Type of change
- [x] Enhancement (improvement to existing features and functionality)

## Checklist
- [ ] Followed the inspiration of the Express.js framework for new functionalities, making them similar in usage.
- [ ] Conducted a self-review of the code and provided comments for complex or critical parts.
- [ ] Updated the documentation in the `/docs/` directory for [Fiber's documentation](https://docs.gofiber.io/).
- [ ] Added or updated unit tests to validate the effectiveness of the changes or new features.
- [ ] Ensured that new and existing unit tests pass locally with the changes.
- [ ] Verified that any new dependencies are essential and have been agreed upon by the maintainers/community.
- [ ] Aimed for optimal performance with minimal allocations in the new code.
- [ ] Provided benchmarks for the new code to analyze and improve upon.

## Commit formatting

Please use emojis in commit messages for an easy way to identify the purpose or intention of a commit. Check out the emoji cheatsheet here: [CONTRIBUTING.md](https://github.com/gofiber/fiber/blob/master/.github/CONTRIBUTING.md#pull-requests-or-commits)

### CODE REVIEW

File: docs/api/ctx.md [hunk 1]
```diff
@@ -384,10 +384,17 @@ type Cookie struct {
...
     SessionOnly bool      `json:"session_only"`
 }
 ```
 
+:::info
+
+Partitioned cookies allow to partition cookie jar by top-level site. You can check out [CHIPS](https://developers.google.com/privacy-sandbox/3pcd/chips) for more information.
+
+:::
+
```
[the remark this answers is not in the document]
Comment (efectn) [reply docs/api/ctx.md:397]: Sure you can

[the remark this answers is not in the document]
Comment (efectn) [reply docs/api/ctx.md:397]: Can you add explanative comment lines for all Cookie properties

**Review by gaby — APPROVED** [review]
LGTM, pending docs updates

**Review by gaby — APPROVED** [review]
LGTM

**Review by ReneWerner87 — APPROVED** [review]

**Pull request merged**